interactions/client/bot.py

```diff
@@ -363,6 +365,55 @@ async def change_presence(self, presence: ClientPresence) -> None:
         """
         await self._websocket._update_presence(presence)
 
+    async def get(
+        self, model: T, http_method: Optional[str] = None, **kwargs: Dict[str, Snowflake]
```

Comment:
```suggestion
        self, model: T, http_method: Optional[str] = None, **kwargs
```

`**kwargs` should never have strict typing involved. If you're trying to do it for documentation purposes, please note it only as being dict-like.

Reply:
Oh I see, will remove